samples/snippets/batch_process_documents_sample_v1beta3.py

```diff
@@ -81,7 +81,7 @@ def batch_process_documents(
         # Download the contents of this blob as a bytes object.
         if ".json" not in blob.name:
```

Comment:
Per our quick GVC, suggest switching to an if else where you check for the positive case
"if .json in blob.name, do the thing"
"else, log the useful info for the end user"

Reply:
Done. Thank you for the speedy review!